src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -126,6 +127,20 @@ const multisigWalletUpdatedFx = createEffect(
   },
 );
 
+type RemoveProps = {
+  wallet: Wallet;
+  accounts: Account[];
```

Comment:
I see that we only require array of `id` from account, let's pass them instead of full accounts 😉